lesson01/main.tf

```diff
@@ -0,0 +1,73 @@
+resource "aws_default_vpc" "default" {
+  tags = {
+    Name = "Default VPC"
+  }
+}
+
+resource "aws_security_group" "web" {
+  name        = "web"
+  description = "Allow web traffic"
+  vpc_id      = "${aws_default_vpc.default.id}"
+
+  ingress {
+    from_port   = 80
+    to_port     = 80
+    protocol    = "TCP"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
```

Comment:
we can move these to a tags variable like this so we can reuse it on all resources
variable "tags" {
  type = "map"

  default = {
  Terraform  = true
    Service    = "web"
    Customer   = "Stark Industries"
    Owner      = "Wizeline"
    Maintainer = "devops@wizeline.com"
  }
}

Reply:
Done!